test/topo/suite.go

```diff
@@ -49,7 +49,7 @@ func (s *TestSuite) SetupTestSuite() error {
 	err = helm.Chart("onos-topo").
 		Release("onos-topo").
 		Set("image.tag", "latest").
-		Set("store.controller", "onos-topo-atomix-kubernetes-controller:5679").
+		Set("storage.controller", "onos-topo-atomix-kubernetes-controller:5679").
```

Comment:
I'm not sure I like the approach here. Can we set up an hour long call to discuss? Tonight at midnight?

Reply:
I think we need to find a less convenient time for @SeanCondon